spotbugs/src/main/java/edu/umd/cs/findbugs/util/ClassName.java

```diff
@@ -265,13 +268,34 @@ public static boolean matchedPrefixes(String[] classSearchStrings, @DottedClassN
         }
 
         for (String p : pp) {
-            if (p.length() > 0 && className.indexOf(p) >= 0) {
+            if (p.length() > 0 && (containsIgnoreCase(className, p) || fuzzyMatch(className, p))) {
                 return true;
             }
         }
 
         return false;
+    }
+
+    private static boolean containsIgnoreCase(String str, String searchStr) {
```

Comment:
If we're using commons-text, then that means that Apache Commons StringUtils is available.

Reply:
Thanks, I did not notice that. I have corrected it to use StringUtils#containsIgnoreCase now!